### ISSUE TITLE
[Request] Ergonomics of HorseMenu

### ISSUE THREAD

**Issue by bunnybuddy213** [YimMenu/HorseMenu#111]
Instead of put hotkeys, a lot of features should be in "Self" and be checked in a box like YimMenu. It would be much simpler
Auto TP in "Teleport" 
etc..

**Comment by Deadlineem** [comment]
I rather like the On/off switch style, it sets it apart from Yim.  However, i do believe things could be more condensed (like having 2 or 3 options on the same line **IF they fit** before the next set of options)

**Comment by bunnybuddy213** [comment]
> I rather like the On/off switch style, it sets it apart from Yim. However, i do believe things could be more condensed (like having 2 or 3 options on the same line **IF they fit** before the next set of options)

Yeah On/Off option is awesome, you're right. I thought Yim was like that as well, long time didnt play haha

**Comment by Rxann** [comment]
> I rather like the On/off switch style, it sets it apart from Yim. However, i do believe things could be more condensed (like having 2 or 3 options on the same line **IF they fit** before the next set of options)

There are limitations to the current rendering system that would need to be fixed for this to be able to happen.

**Comment by Deadlineem** [comment]
> > I rather like the On/off switch style, it sets it apart from Yim. However, i do believe things could be more condensed (like having 2 or 3 options on the same line **IF they fit** before the next set of options)
> 
> Yeah On/Off option is awesome, you're right. I thought Yim was like that as well, long time didnt play haha I mostly wanted to talk about the layout.

The layout could indeed use some more work, in due time though.
